base/uk.ac.stfc.isis.ibex.opis/resources/lakeshore340.opi

```diff
@@ -17,21 +17,21 @@ from org.csstudio.opibuilder.scriptUtil import ConsoleUtil
   </auto_scale_widgets>
   <auto_zoom_to_fit_all>false</auto_zoom_to_fit_all>
   <background_color>
-    <color name="ISIS_OPI_Background" red="240" green="240" blue="240"/>
+    <color name="ISIS_OPI_Background" red="240" green="240" blue="240" />
   </background_color>
-  <boy_version>5.1.0.201707071649</boy_version>
+  <boy_version>5.1.0</boy_version>
   <foreground_color>
-    <color name="ISIS_OPI_Foreground" red="192" green="192" blue="192"/>
+    <color name="ISIS_OPI_Foreground" red="192" green="192" blue="192" />
   </foreground_color>
   <grid_space>6</grid_space>
   <height>600</height>
   <macros>
     <include_parent_macros>true</include_parent_macros>
-    <PV_ROOT>$(P)$(LKSH340)</PV_ROOT>
+    <PV_ROOT>$(P)LKSH340_01</PV_ROOT>
```

Comment:
This doesn't look right - it means we can never have a `LKSH340_02`?

Reply:
My bad, set it for testing and forgot to switch it back